app/views/partials/theme/simple/items/show.liquid

```diff
@@ -88,3 +110,53 @@
     </div>
   </div>
 </div>
+
+
+<div class="container">
+  <p class="text-2xl text-gray-900 ">Reviews</p>
+  <div class="recent-reviews">
+    {% for review in item.reviews %}
+      <div class="relative py-2 border-t border-blue-300">
+        <div class="flex flex-wrap mb-4">
+          <div class="rating">
+            <div class="rating-point" style="width: {{ review.rating | divided_by:5 | times:100 }}%"></div>
+          </div>
+          <span class="rating-score">{{ review.rating | round: 2 }}</span>
+        </div>
+        <p><b>{{ review.owner}}</b> : {{ review.comment }}</p>
+      </div>
+    {% endfor %}
+  </div>
+  
+  {%- if context.current_user.id %}
```

Comment:
In terms of functionality: you probably want to ensure the user has not already added a review to this product

In terms of code architecture: We would like to avoid using `context` directly in views; the suggested way is to provide all necessary variables in Page, i.e. `app/views/pages/items.liquid`